src/templates/pt/2019/base.html

```diff
@@ -0,0 +1,229 @@
+  maneira clara e correta de fazer as coisas. Com a ajuda contínua da comunidade da web, esperamos fazer disso uma
+  tradição anual, para que a cada ano possamos acompanhar nosso progresso e fazer as correções necessárias, conforme
+  necessário.
+</p>
+
+<p>
+  Há muito o que aprender neste relatório. Portanto, comece a explorar e compartilhar suas sugestões com a comunidade
+  para que possamos avançar coletivamente em nossa compreensão do estado da web.
+</p>
+
+<p>— <em><a href="{{ url_for('contributors', year=year, lang=lang, _anchor='rviscomi') }}">Rick Viscomi</a>, criador do
+    Web Almanac</em></p>
+{% endblock %}
+
+{% block appendix %}Apêndice{% endblock %}
+{% block appendices %}Apêndices{% endblock %}
+
+{% block ebook_title %}Ebook{% endblock %}
+{% block ebook_download %}Faça o download do Web Almanac {{ year }} completo em formato PDF{% endblock %}
+{% block ebook_download_note %}(gerado em <a href="https://www.princexml.com/">www.princexml.com</a>){% endblock %}
+
+{% 
+  set localizedPartTitles = {
+    "I. Page Content": "I. Conteúdo da Página",
+    "II. User Experience": "II. Experiência de usuário",
+    "III. Content Publishing": "III. Publicação de conteúdo",
+    "IV. Content Distribution": "IV. Distribuição de Conteúdo"
+  }
+%}
+{%
+  set localizedChapterTitles = {
+    "javascript": "JavaScript",
+    "css": "CSS",
+    "markup": "Marcação",
+    "media": "Mídia",
+    "third-parties": "Terceiros",
+    "fonts": "Fontes",
+    "performance": "Desempenho",
+    "security": "Segurança",
+    "accessibility": "Acessibilidade",
+    "seo": "SEO",
+    "pwa": "PWA",
+    "mobile-web": "Web móvel",
+    "ecommerce": "Comércio eletrônico",
+    "cms": "CMS",
+    "compression": "Comércio eletrônico",
+    "caching": "Armazenamento em Cache",
+    "cdn": "CDN",
+    "page-weight": "Peso da página",
+    "resource-hints": "Dicas de Recursos",
+    "http2": "HTTP/2"
+  }
+%}
+
+{% set localizedTeamNames = {
+  "analysts": "Analistas",
+  "authors": "Autores",
+  "brainstormers": "Pensadores",
+  "designers": "Designers",
+  "developers": "Desenvolvedores",
+  "editors": "Editores",
+  "leads": "Lider",
+  "reviewers": "Revisores",
+  "translators": "Tradutores",
+  }
+%}
+
+{% set localizedContributors = {
+  "kleinab": 'Abigail Klein is a Google software engineer. She worked on Google Docs, Sheets, and Slides web accessibility where she added <a href="https://www.blog.google/outreach-initiatives/accessibility/whats-you-say-present-captions-google-slides/">automatic captions to Google Slides</a>, as well as improving screen reader, braille, screen magnifier, and high contrast support. She currently works on Google Chrome and ChromeOS accessibility. She has a bachelor\'s and master\'s degree in computer science from MIT, where she co-founded an assistive technology hackathon and was a lab assistant and guest lecturer of the assistive technology class.',
```

Comment:
Can we translate bios these too?

Reply:
@bazzadp Sorry I'm late.

I made the suggested changes, could you guide me on how to send the new update please?